bash_completion

```diff
@@ -917,7 +917,7 @@ _mac_addresses()
     # - ip link: link/ether
     COMPREPLY+=($(
         {
-            LC_ALL=C ifconfig -a || ip link show
+            LC_ALL=C ifconfig -a || (ip -c=never link show || ip link show)
```

Comment:
Please remove the parenthesis from everywhere, they're not needed, and actually start an unnecessary subshell.